.vscode/settings.json

```diff
@@ -464,6 +464,15 @@
         "__split_buffer": "cpp",
         "__threading_support": "cpp",
         "__tree": "cpp",
-        "print": "cpp"
+        "print": "cpp",
+        "__bits": "cpp",
+        "__debug": "cpp",
+        "__errc": "cpp",
+        "__functional_base": "cpp",
+        "__mutex_base": "cpp",
+        "__nullptr": "cpp",
+        "__std_stream": "cpp",
+        "__string": "cpp",
+        "__tuple": "cpp"
```

Comment:
Let me know if I should revert this, I did not knowingly make any changes here and am unaware of what the impact of these might be.

Reply:
Don't worry about these, I dunno what the mechanic is that adds stuff to this, but it happens from time to time 🤷